src/index.ts

```diff
@@ -631,11 +696,11 @@ export class Bigtable {
         client: 'BigtableInstanceAdminClient',
         method: 'listInstances',
         reqOpts,
-        gaxOpts: gaxOptions,
+        gaxOpts: gaxOptionsOrCallback,
```

Comment:
Once you introduce `gaxOptions` variable as per comment above, this change will not be needed.